img/a/addy.io.svg

```diff
@@ -0,0 +1,95 @@
+				c-1.3-0.5-2.4-1.2-3.3-2.3c-0.9-1.1-1.3-2.3-1.3-3.8c0-2.2,0.8-3.8,2.4-4.9c1.6-1.1,4-1.6,7.3-1.6c2.3,0,4.5,0.3,6.5,0.8
+				c0.9,0.2,1.7,0.5,2.4,0.8v8C187.5,130.9,186.7,131.9,185.2,132.4z"/>
+            <path class="st4" d="M240.1,94.9c-0.6-0.3-1.3-0.6-2-0.9c-1.7-0.7-3.4-1.1-5.1-1.3c-1.7-0.2-3.1-0.3-4.1-0.3
+				c-7.4,0-13,2.2-16.6,6.5c-3.7,4.3-5.5,10.2-5.5,17.7c0,5,0.6,9.1,1.9,12.4c1.2,3.3,3,5.8,5.2,7.7c2.2,1.8,4.8,3.1,7.7,3.9
+				c2.9,0.8,6,1.2,9.2,1.2c2.6,0,5.1-0.3,7.5-0.9c2.5-0.6,4.7-1.3,6.7-2.3c2-0.9,3.7-2.2,4.9-3.8c1.3-1.6,1.9-3.4,1.9-5.5v-55h-11.7
+				V94.9z M240.1,128.2c0,1.1-0.8,2-2.5,2.6c-1.7,0.7-3.8,1-6.3,1c-3.2,0-5.7-0.7-7.6-2c-1.8-1.3-3.1-3.1-3.8-5.3s-1.1-4.8-1.1-7.7
+				c0-3.4,0.4-6.2,1.3-8.4c0.9-2.2,2.3-3.8,4.2-4.8c1.9-1,4.2-1.5,7-1.5c1.4,0,2.9,0.2,4.4,0.6c1.6,0.4,3,1,4.4,1.8V128.2z"/>
+            <path class="st4" d="M292.7,94.9c-0.6-0.3-1.3-0.6-2-0.9c-1.7-0.7-3.4-1.1-5.1-1.3s-3.1-0.3-4.1-0.3c-7.4,0-13,2.2-16.6,6.5
+				c-3.7,4.3-5.5,10.2-5.5,17.7c0,5,0.6,9.1,1.9,12.4c1.2,3.3,3,5.8,5.2,7.7c2.2,1.8,4.8,3.1,7.7,3.9c2.9,0.8,6,1.2,9.2,1.2
+				c2.6,0,5.1-0.3,7.5-0.9c2.5-0.6,4.7-1.3,6.7-2.3c2-0.9,3.7-2.2,4.9-3.8c1.3-1.6,1.9-3.4,1.9-5.5v-55h-11.7V94.9z M292.7,128.2
+				c0,1.1-0.8,2-2.5,2.6c-1.7,0.7-3.8,1-6.3,1c-3.2,0-5.7-0.7-7.6-2c-1.8-1.3-3.1-3.1-3.8-5.3s-1.1-4.8-1.1-7.7
+				c0-3.4,0.4-6.2,1.3-8.4c0.9-2.2,2.3-3.8,4.2-4.8c1.9-1,4.2-1.5,7-1.5c1.4,0,2.9,0.2,4.4,0.6c1.6,0.4,3,1,4.4,1.8V128.2z"/>
+            <path class="st4" d="M346.6,93.7l-11.2,32.7l-11.3-32.7h-13.4l18.8,46.3c-0.3,1.1-0.7,2.1-1,3c-0.8,1.7-1.8,3-3,3.9
+				c-1.2,0.9-2.6,1.3-4.3,1.3c-0.9,0-1.8-0.1-2.6-0.3c-0.8-0.1-1.6-0.4-2.5-0.7v9.8c0.9,0.4,2,0.7,3.3,1c1.2,0.3,2.5,0.4,3.9,0.4
+				c2,0,4-0.5,6-1.5c2-0.9,3.9-2.6,5.8-5c1.8-2.4,3.6-5.7,5.2-9.8l18.8-48.4H346.6z"/>
+		</g>
+
+            <linearGradient id="SVGID_00000056422304933101961500000013808243367702213549_" gradientUnits="userSpaceOnUse" x1="366.4" y1="77.8111" x2="366.4" y2="64.9222" gradientTransform="matrix(1 0 0 -1 0 206)">
+			<stop  offset="0" style="stop-color:#3AE7E1"/>
+                <stop  offset="0.9481" style="stop-color:#1993D2"/>
+		</linearGradient>
+            <path style="fill:url(#SVGID_00000056422304933101961500000013808243367702213549_);" d="M360.3,140.4v-11.9h12.2v11.9H360.3z"/>
+            <path class="st4" d="M386.8,86.7V77h12.3v9.7H386.8z M387.1,140.4V94.2h11.8l0.1,46.2H387.1z"/>
+            <path class="st4" d="M429.8,141.8c-7.7,0-13.5-2.1-17.3-6.4c-3.8-4.3-5.7-10.4-5.7-18.4c0-8.6,2-14.9,5.9-18.8
+			c3.9-3.9,9.6-5.8,17.1-5.8c5.2,0,9.5,0.9,12.9,2.7c3.4,1.8,6,4.6,7.7,8.2s2.6,8.2,2.6,13.7c0,8-2,14.2-6,18.4
+			C443,139.6,437.3,141.8,429.8,141.8z M429.8,132.1c2.7,0,4.9-0.6,6.6-1.9c1.7-1.2,2.9-3,3.7-5.3c0.8-2.3,1.2-5,1.2-8.1
+			c0-3.5-0.4-6.4-1.2-8.6c-0.8-2.2-2.1-3.9-3.8-4.9c-1.7-1-3.9-1.6-6.5-1.6c-2.7,0-4.9,0.6-6.6,1.7s-2.9,2.8-3.6,5s-1.1,5-1.1,8.4
+			c0,4.9,0.9,8.6,2.6,11.3C422.9,130.8,425.7,132.1,429.8,132.1z"/>
+	</g>
+</g>
+</svg>
```

Comment:
[SVGOMG](https://jakearchibald.github.io/svgomg/) is a good tool for minifying SVGs.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 83.8 112"><defs><linearGradient id="a" x1="42.7" x2="130.7" y1="135.8" y2="47.8" gradientTransform="matrix(1 0 0 -1 0 206)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#3ae7e1"/><stop offset=".9" stop-color="#1993d2"/></linearGradient><linearGradient id="c" x1="42.7" x2="130.7" y1="135.7" y2="47.7" gradientTransform="matrix(1 0 0 -1 0 206)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#3ae7e1"/><stop offset=".9" stop-color="#1993d2"/></linearGradient><linearGradient id="b" x1="67.8" x2="155.7" y1="160.8" y2="72.8" gradientTransform="matrix(1 0 0 -1 0 206)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#3ae7e1"/><stop offset=".9" stop-color="#1993d2"/></linearGradient></defs><path fill="#2d7aae" d="M69.9 63c0 5.2-2.8 9.7-7 12.1V84a7 7 0 0 1-7 7 28 28 0 0 0 13.5-3.5c.3-1.1.5-2.3.5-3.5z"/><path fill="url(#a)" d="M102.9 95.5H75a7 7 0 0 0-7 7v28a7 7 0 0 0 7 7h28a7 7 0 0 0 7-7v-28c-.1-3.9-3.2-7-7.1-7zm-7 28H82v-14h14z" transform="translate(-47 -46.5)"/><path fill="url(#b)" d="M76.9 67.5c2.4-4.2 6.9-7 12.1-7 5.2 0 9.7 2.8 12.1 7h1.9c4.7 0 9.3 1 13.6 2.8a27.9 27.9 0 0 0-55.2 0c4.3-1.8 8.9-2.8 13.6-2.8Z" transform="translate(-47 -46.5)"/><path fill="#2d7aae" d="M14 31.8c4.1-2.4 8.9-3.8 14-3.8 0-2.5.7-4.9 1.9-7H28c-4.7 0-9.3 1-13.6 2.8l-.3 4.2v3.8zM54 21c1.2 2.1 1.9 4.4 1.9 7 5.1 0 9.9 1.4 14 3.8V28l-.3-4.2A35.2 35.2 0 0 0 56 21z"/><path fill="url(#c)" d="M116.9 130.5c0 1.2-.2 2.4-.5 3.5a28 28 0 0 0 14.4-24.5v-7a27.9 27.9 0 0 0-28-28H75c-5.1 0-9.9 1.4-14 3.8a27.7 27.7 0 0 0-14 24.2v28a28 28 0 0 0 28 28h28a14 14 0 0 0-14-14H75a14 14 0 0 1-14-14v-28a14 14 0 0 1 14-14h28a14 14 0 0 1 14 14v7z" transform="translate(-47 -46.5)"/></svg>
```